.github/workflows/ci.yml

```diff
@@ -55,26 +58,24 @@ jobs:
     - name: Tests
       if: ${{ !cancelled() && steps.deps.conclusion == 'success' }}
       run: |
-        coverage run runtests.py --parallel
```

Comment:
Do we have to do this or can we avoid doing it? I thought you found a way to resolve the coverage issue by adding `thread` to the multiprocessing options.